### PR TITLE
aruha-694: HIlaAT.whenStreamTimeoutReachedPossibleToCommit fix

### PR DESCRIPTION
**Pull request by adyach**
This PR bring fix for failing from time to time test, the problem was in the implementation of the streaming client, which was hanging while closing the stream, but the connection was closed from Nakadi side and commit timeout expired thus it was not possible to commit cursors.
image

### CODE REVIEW

File: src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java [hunk 1]
```diff
@@ -117,11 +115,8 @@ public TestStreamingClient start() {
 
     public boolean close() {
         if (running) {
-            try {
-                connection.disconnect();
-            } finally {
-                running = false;
-            }
+            running = false;
+            connection.disconnect();
```
Comment (v-stepanov) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: @adyach have you discussed that with @antban? I know that in his branch he changed the way of closing connection by closing connection of URL object.

Comment (adyach) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: I think he already pushed it to master, but it did not help. The `connection` is the instance of the `HttpUrlConnection`. Is that what you meant?

Comment (v-stepanov) [inline src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java]: Ah ok, I see now. I took a look at his branch - yes he actually closes HttpUrlConnection. So everything is ok.

**Comment by antban** [comment]
:+1:

**Pull request merged**